tests/fe/fe_nedelec_sz_hanging_nodes_convergence.cc

```diff
@@ -0,0 +1,448 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+// By Sebastian Kinnewig
```

Comment:
Can you please remove this line? We avoid tagging files with particular authors as we see the library as owned and maintained by a collective of all deal.II authors, including you. Your contribution will obviously still be visible via the commit author/git annotation.